rocketpool-cli/node/status.go

```diff
@@ -312,6 +312,19 @@ func getStatus(c *cli.Context) error {
 		fmt.Println("The node is not registered with Rocket Pool.")
 	}
 
+	// Alerts
+	if cfg.EnableMetrics.Value == true {
+		// only print alerts if enabled; to avoid misleading the user to thinking everything is fine (since we really don't know).
+		fmt.Printf("\n%s=== Alerts ===%s\n", colorGreen, colorReset)
+		for _, alert := range status.Alerts {
+			fmt.Println(alert.ColorString())
```

Comment:
I'm worried this pollutes the status page. Would rather see somethings like "There are X alerts enabled", and then a specific command to list them.

Reply:
pretty sure it only prints alerts that are actively firing.

we should print the `=== Alerts ===` conditionally on there being any firing alerts though

Reply:
I removed alerts section when no alerts are firing and limited it to 3 alerts in ecb830c5cd66e78fa1daabc10f67e06763882259